python/lsst/pipe/base/_dataset_handle.py

```diff
@@ -43,32 +50,36 @@ class InMemoryDatasetHandle:
     parameters will be converted into a dataId-like entity.
     """
 
+    _empty = DataCoordinate.makeEmpty(DimensionUniverse())
```

Comment:
You said on the ticket that this is a slow operation.  How often is this run now that it's in the class?

Reply:
Once. It's a class property so defined at class definition. @TallJimbo I do wonder why `makeEmpty` is so slow -- `DimensionUniverse()` is meant to cache so I wasn't expecting there to be overhead in that call but maybe there is overhead I wasn't expecting and caching the universe would have been enough.

Reply:
Looks like the caching remembers all of the `DimensionUniverse` instances that exist, but it doesn't record which one is the default, and I think that means it has to go read the default config file to find at least the version of the default one before it can look for it in the cache.  We could clearly make that more efficient.